src/lib/nullifier.ts

```diff
@@ -165,4 +170,63 @@ class Nullifier extends Struct({
   getPublicKey() {
     return PublicKey.fromGroup(this.publicKey);
   }
+
+  /**
+   *
+   * _Note_: This is *not* the recommended way to create a Nullifier in production. Please use mina-signer to create Nullifiers. Also, this function cannot be run within provable code.
```

Comment:
Could you explain briefly why you can't run this in provable code, if it's possible? What are the limitations so devs can know why they shouldn't do this.